### PR TITLE
vcn feature

### PR DESCRIPTION
**Pull request by AmyBeall**

### CODE REVIEW

File: AffirmSDK/AffirmCreditCard.m [hunk 1]
```diff
@@ -48,12 +44,17 @@ - (instancetype)initWithDict:(NSDictionary *)dict
...
-        _cardholderName = dict[@"cardholder_name"];
-        _expiration = dict[@"expiration"];
         _creditCardId = dict[@"id"];
+        _callbackId = dict[@"callback_id"];
+        _cvv = nil;
+        _number = nil;
+        _cardholderName = nil;
+        _expiration = nil;
+        
+        if (dict[@"cvv"]) _cvv = dict[@"cvv"];
```
Comment (bf-affirm) [inline AffirmSDK/AffirmCreditCard.m]: Although this works, if these instance variables are nullable, you can just assign from the dict

**Review by bf-affirm — COMMENTED** [review]
ok, but see comment

File: AffirmSDK/AffirmCreditCard.m (hunk 1 above)
Comment (AmyBeall) [reply AffirmSDK/AffirmCreditCard.m]: Thank you!

Comment (AmyBeall) [reply AffirmSDK/AffirmCreditCard.m]: @bf-affirm Fixed!

**Review by bf-affirm — APPROVED** [review]

**Review by bf-affirm — APPROVED** [review]

**Review by bf-affirm — APPROVED** [review]

**Pull request merged**